.github/workflows/workflow_test.yml

```diff
@@ -44,6 +44,14 @@ jobs:
         mode: lint
         workflows: true
         repository-list: "${{ needs.setup.outputs.repository-list }}"
+    - name: IWC specific lint
+      run: |
+        for REPO in ${{ needs.setup.outputs.repository-list }}; do
```

Comment:
This does not work. When the repository-list is more than one it is replaced by the different values separated by back to line character, for example:
```
for REPO in workflows/blabla/blibli
workflow/blabla/blublu; do
```
And this is not a valid bash command.